src/main/java/entities/User.java

```diff
@@ -1,4 +1,37 @@
 package entities;
 
+import java.util.List;
+import java.util.HashMap;
+import java.util.ArrayList;
+
+// import other required classes
+// import list of cuisine categories (?)
+
+/**
+ * User class stores: username, password, HashMap mapping cuisine to number of likes (Integer) user put for that
+ * cuisine (String), a String bio, a list of Users that follow User (followers), a list of Users the user is following,
+ * a list of Posts the user has posted
+ */
 public class User {
+    public String username, password, bio;
```

Comment:
All these instance variables should probably be private + have getters and setters

Reply:
agreed